test/service/test_jobs.py

```diff
@@ -330,15 +330,15 @@ def test_logs_creation_failure_during_database_insert(self):
         ], logstream.getvalue().splitlines())
 
     def test_throws_when_piazza_throws(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.